version.json

```diff
@@ -1,104 +1,104 @@
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-common": {
             "package": "shared/AppInsightsCommon/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-core-js": {
             "package": "shared/AppInsightsCore/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-offlinechannel-js": {
             "package": "channels/offline-channel-js/package.json",
-            "release": "0.0.1"
+            "release": "0.1.0"
         },
         "@microsoft/applicationinsights-chrome-debug-extension": {
             "package": "tools/chrome-debug-extension/package.json",
-            "release": "0.4.8"
+            "release": "0.5.0"
         },
         "applicationinsights-web-config": {
             "package": "tools/config/package.json",
-            "release": "0.0.12"
+            "release": "0.1.0"
         },
         "@microsoft/applicationinsights-example-aisku": {
             "package": "examples/AISKU/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-example-dependencies": {
             "package": "examples/dependency/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-example-shared-worker": {
             "package": "examples/shared-worker/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-teechannel-js": {
             "package": "channels/tee-channel-js/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-test-module-type-check": {
             "package": "AISKU/Tests/es6-module-type-check/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-cfgsync-js": {
             "package": "extensions/applicationinsights-cfgsync-js/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/applicationinsights-example-cfgsync": {
             "package": "examples/cfgSync/package.json",
-            "release": "3.0.8"
+            "release": "3.1.0"
         },
         "@microsoft/1ds-post-js": {
             "package": "channels/1ds-post-js/package.json",
-            "release": "4.0.6"
+            "release": "4.1.0"
         },
         "@microsoft/1ds-core-js": {
             "package": "shared/1ds-core-js/package.json",
-            "release": "4.0.6"
+            "release": "4.1.0"
         },
         "@microsoft/applicationinsights-web-snippet": {
             "package": "tools/applicationinsights-web-snippet/package.json",
-            "release": "1.1.6"
+            "release": "1.1.0"
```

Comment:
The last released version of this package was 1.0.1, so rolling back this number to the next release